src/views/month/titlebar.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Month View Title Template
+ * The title template for the month view of events.
+ *
+ * Override this template in your own theme by creating a file at [your-theme]/tribe-events/month/titlebar.php
+ *
+ * @package TribeEventsCalendar
+ *
```

Comment:
Lets use `month/title-bar` for the name of this file.
Also lets add `@version TBD` and a `@since TBD` that `TBD` is a placeholder for us to replace on release time. Just to flag this was changed.

Reply:
Fixed.